docs/_docs/contributing.md

```diff
@@ -71,6 +71,26 @@ You can find the documentation for jekyllrb.com in the [docs](https://github.com
 
 One gotcha, all pull requests should be directed at the `master` branch (the default branch).
 
+### Updating FontAwesome iconset for jekyllrb.com
+
+We recently moved to using a stripped-down version of FontAwesome iconset on the site, consisting of only those icons that we actually use here.
```

Comment:
Maybe just "We use a custom version of FontAwesome which contains just the icons we use." 4 years from now this will still say "recently" :laughing: